include/gatekeeper_net.h

```diff
@@ -242,6 +246,20 @@ extern uint8_t rss_key_be[RTE_DIM(default_rss_key)];
 		0xFF, ipv6[13], ipv6[14], ipv6[15],	\
 	}
 
+static inline int
+lacp_enabled(struct net_config *net, struct gatekeeper_if *iface)
+{
+	/* When @iface is the back, need to make sure it's enabled. */
+	if (iface == &net->back)
+		return net->back_iface_enabled &&
+			iface->num_ports > 1 &&
```

Comment:
Ah, I see. Just to confirm: this means that even when there is a single port, we should create a bonded port if LACP is being used (in contrast to the current initialization, which does not do round-robin bonding on the port at all if there's only one).